engine/src/main/java/org/camunda/bpm/engine/impl/batch/deletion/DeleteHistoricProcessInstanceBatchConfigurationJsonConverter.java

```diff
@@ -16,12 +16,15 @@
  */
 package org.camunda.bpm.engine.impl.batch.deletion;
 
+import java.util.List;
+
 import org.camunda.bpm.engine.impl.batch.BatchConfiguration;
+import org.camunda.bpm.engine.impl.batch.DeploymentMappingJsonConverter;
+import org.camunda.bpm.engine.impl.batch.DeploymentMappings;
 import org.camunda.bpm.engine.impl.json.JsonObjectConverter;
 import org.camunda.bpm.engine.impl.util.JsonUtil;
-import com.google.gson.JsonObject;
 
-import java.util.List;
```

Comment:
Can we try to avoid reorganising imports?

Reply:
I changed my IDE setup a bit to avoid automatic reorganization in more situations, thanks for the hint. Would you like me to revert some of the import reorganizations that already happened in this PR? If that helps with the PR, I can surely do that.

Reply:
No need to change this within this PR. Let's keep it in mind for the future.

Reply:
Will do 👍